src/poliastro/tests/tests_twobody/test_orbit.py

```diff
@@ -968,25 +968,30 @@ def test_from_coord_if_coord_is_not_of_shape_zero():
 
 @pytest.mark.remote_data
 def test_from_sbdb():
-    # Dictionary with structure: 'Object': [a, e, i, raan, argp, nu, epoch]
-    # Notice JPL provides Mean anomaly, a conversion is needed to obtain nu
-    SBDB_DATA = {
-        "Ceres": (
-            2.76916515450648 * u.AU,
-            0.07600902910070946 * u.one,
-            10.59406704424526 * u.deg,
-            80.30553156826473 * u.deg,
-            73.597694115971 * u.deg,
-            86.01851107780747 * u.deg,
-        )
-    }
+    targets = ["Ceres", "Vesta", "Eros"]  # Objects in both JPL SBDB and JPL Horizons
```

Comment:
Let's replace this with `@pytest.mark.parametrize`, as done in other tests

Reply:
Good idea - added parametrization. I left the remote decorator in as well. I think python will apply both decorators from bottom to top, so this should be ok.